src/components/Alert/Alert.stories.tsx

```diff
@@ -1,6 +1,9 @@
 import { Meta, Story } from '@storybook/react/types-6-0';
 
+import React from 'react';
+
 import { Alert, AlertProps } from '.';
+import { HiInformationCircle, HiEye } from 'react-icons/hi';
```

Comment:
Can we move this import under the storybook one?

Reply:
Sorry, I don't get it. What do you mean by that?

Reply:
I meant if we can move this line to line `2`